cli/create_test.go

```diff
@@ -416,6 +416,60 @@ func TestCreateWithRichParameters(t *testing.T) {
 		assert.ErrorContains(t, err, "parameter \""+wrongFirstParameterName+"\" is not present in the template")
 		assert.ErrorContains(t, err, "Did you mean: "+firstParameterName)
 	})
+
+	t.Run("CopyParameters", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+		owner := coderdtest.CreateFirstUser(t, client)
+		member, _ := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID)
+		version := coderdtest.CreateTemplateVersion(t, client, owner.OrganizationID, echoResponses)
+		coderdtest.AwaitTemplateVersionJobCompleted(t, client, version.ID)
+
+		template := coderdtest.CreateTemplate(t, client, owner.OrganizationID, version.ID)
+
+		// Firstly, create a regular workspace using template with parameters.
+		inv, root := clitest.New(t, "create", "my-workspace", "--template", template.Name, "-y",
```

Comment:
NB: we now have [dbfake.WorkspaceWithAgent](https://github.com/coder/coder/blob/main/coderd/database/dbfake/dbfake.go#L42) if you want to speed things up

Reply:
`WorkspaceWithAgent` looks efficient, but I need a workspace with build parameters and a corresponding template with rich parameters. Not sure if the function supports it.

Reply:
😢 🎺